emailTemplates/newParticipantReadyForReview.hbs

```diff
@@ -1,54 +1,55 @@
 <html>
   <head>
-    <meta http-equiv='Content-Type' content='text/html; charset=utf-8' />
-    <meta
-      name='viewport'
-      content='width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1'
-    /><!--[if !mso]><!-->
-    <meta http-equiv='X-UA-Compatible' content='IE=Edge' /><!--<![endif]-->
-    <!--[if (gte mso 9)|(IE)]>
-    <xml>
-    <o:OfficeDocumentSettings>
-    <o:AllowPNG/>
-    <o:PixelsPerInch>96</o:PixelsPerInch>
-    </o:OfficeDocumentSettings>
-    </xml>
-    <![endif]-->
-    <!--[if (gte mso 9)|(IE)]>
-    <style type="text/css">
-      body {width: 600px;margin: 0 auto;}
-      table {border-collapse: collapse;}
-      table, td {mso-table-lspace: 0pt;mso-table-rspace: 0pt;}
-      img {-ms-interpolation-mode: bicubic;}
-    </style>
-    <![endif]-->
+    <link rel='preconnect' href='https://fonts.googleapis.com' />
+    <link rel='preconnect' href='https://fonts.gstatic.com' crossorigin />
+    <link href='https://fonts.googleapis.com/css2?family=Inter&display=swap' rel='stylesheet' />
     <style type='text/css'>
-      body, p, div { padding: 70px; text-align: center; font-family: "Inter"; font-size: 18px; }
+      body { padding: 70px; text-align: center; font-family: 'Inter', sans-serif; font-size: 18px; }
       body { color: #000000; background-color: #FFF; } .logo-container { padding: 20px } .logo {
-      height: 30px; } .content { margin-top: 70px; width:500px; display: inline-block; } .button {
-      border-radius: 5px; font-weight: 700; font-size: 18px; padding: 15px 120px 15px 120px; margin:
-      0 auto; background-color: #cdf200; color: #030a40; width: auto; text-align: center;
-      text-decoration: none; } .content-text { text-align: left; }
+      height: 30px; } .content { margin-top: 50px; width:500px; display: inline-block; color:
+      #030A40} .button { color: #0459FD; border-radius: 5px; font-weight: 700; font-size: 18px;
+      padding: 15px 120px 15px 120px; margin: 0 auto; width: auto; text-align: center;
+      text-decoration: none; } .content-text { text-align: left; font-size: 18px; }
+      .participant-detail { background: rgba(192, 214, 255, 0.3); border-radius: 5px; padding: 16px
+      22px; margin-top: 24px;} .table-header { text-align: left; font-weight: 600; font-size: 16px;
+      line-height: 29px; color: #000000; } table { text-align: left; font-weight: 400; font-size:
+      14px; line-height: 34px; } tbody tr td:first-child { width: 250px; min-width: 250px;
+      max-width: 250px; word-break: break-all; } .table-header:nth-child(3) { margin-top: 24px; }
+      .footer { margin-top: 32px; }
     </style>
-
   </head>
   <body>
-    <div class='logo-container'>
+    <div>
       <a href='http://localhost:3000/'><img
```

Comment:
how would these localhost references be resolved in integration/prod environemtns?

Reply:
We have a separate ticket to update the logo in templates. I've add this file in the ticket description as well. https://atlassian.thetradedesk.com/jira/browse/UID2-985